validator/tests/unit/test_signed_object.py

```diff
@@ -78,8 +76,7 @@ def test_signed_node(self):
         # Verify that signed_node and sign_object does not invalidate the
         # signed object and can be returned to original
         # create initial signed object
-        signkey = SigObj.generate_signing_key()
-        temp = SignedObject({signkey: "test"}, signkey)
+        temp = SignedObject({"TestSignatureName": "test"}, "TestSignatureName")
```

Comment:
I didn't simplify any of these tests, I just fixed the misunderstanding use of the ```signkey``` replaced by ```SignatureName``` attribute. So yes and I checked, I didn't break anything.